syntax = "proto3";
package it.smaso.drawingtablet;

import "google/protobuf/empty.proto";

message ScreenSize {
  int32 width = 1;
  int32 height = 2;
}

message Coordinates {
  float x = 1;
  float y = 2;
}

message CursorPosition{
  bool clicking = 1;
  Coordinates coordinates = 2;
}

service MainService {
  rpc GetScreenSize(google.protobuf.Empty) returns (ScreenSize);
  rpc OnCursorPosition(CursorPosition) returns (google.protobuf.Empty);
}